### ISSUE TITLE
Add option for label on input field.

### ISSUE THREAD

**Issue by MathiasaurusRex** [the-couch/rimp#2]
Great project, thanks for publishing it!

I would like to add an option for a label within the component so that it better passes accessibility best practices as defined by deque/axe ( https://dequeuniversity.com/rules/axe/2.2/label?application=lighthouse )

This is partially for my own self-interest -- the only item that was dinged on my own accessibility audit was the lack of form label on the Rimp form component.

I haven't contributed much to open source on Github but if you think this would add some value I could take a stab at it!

Thanks!

**Comment by iamkevingreen** [comment]
@MathiasaurusRex more than happy for you to take a stab at it, we'll most likely refactor this as a simple form submit wrapper rather than influencing the actual markup as well.

**Comment by iamkevingreen** [comment]
@MathiasaurusRex merged in and pushed up to npm
